.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
                   python-version: ${{ matrix.python-version }}
             - run: |
                 sudo apt update
-                sudo apt install graphviz imagemagick x11-apps xserver-xephyr xterm xvfb libpulse-dev lm-sensors
+                sudo apt install libdbus-1-dev libgirepository1.0-dev gir1.2-gtk-3.0 gir1.2-notify-0.7 gir1.2-gudev-1.0 graphviz imagemagick x11-apps xserver-xephyr xterm xvfb libpulse-dev lm-sensors
```

Comment:
+1 to this, if you make a separate PR I'm happy to take it.

Reply:
yeah, we weren't actually doing static analysis on large sections of the code because of these missing libraries.

Reply:
I can make this a separate PR I thnk

Reply:
As I noted in IRC, actually this is in another PR: https://github.com/qtile/qtile/pull/2102

Reply:
But that has the composable requirements as well :). Any chance we can get this alone?

Reply:
done https://github.com/qtile/qtile/pull/2162